spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -165,7 +166,7 @@ public boolean parquetVectorizationEnabled() {
         .parse();
   }
 
-  public int parquetBatchSize() {
+  private int parquetBatchSize() {
```

Comment:
We probably shouldn't make these methods private for now. Our `SparkReadConf` is fairly public, so we should try avoid breaking it.